### PR TITLE
At the end of drag and drop, grab and release seat

### PR DESCRIPTION
**Pull request by zhuyaliang**
Fix https://github.com/mate-desktop/engrampa/issues/515

### CODE REVIEW

**Comment by zhuyaliang** [comment]
@cwendling Do you accept this repair method

**Comment by zhuyaliang** [comment]
@cwendling Looking forward to your review

File: src/fr-window.c [hunk 1]
```diff
@@ -4160,6 +4160,25 @@ file_list_drag_begin (GtkWidget          *widget,
...
+    GdkDisplay *display;
+    GdkSeat *seat;
+
+    g_return_if_fail (widget != NULL);
+    window = gtk_widget_get_window (widget);
+    display = gdk_window_get_display (window);
+
+    seat = gdk_display_get_default_seat (display);
+    gdk_seat_grab (seat, window,
+                   GDK_SEAT_CAPABILITY_ALL, TRUE,
```
Comment (cwendling) [inline src/fr-window.c:4177]: That's probably sufficient
```suggestion
                   GDK_SEAT_CAPABILITY_ALL_POINTING, TRUE,
```

**Comment by cwendling** [comment]
@zhuyaliang I don't like the "fix", but at least it seems to work.

The root issue seems to be inside *src/eggtreemultidnd.c*'s `egg_tree_multi_drag_motion_event()`, or possibly in how pcmanfm handles drag events.
IIUC it comes from pcmanfm requesting the drag data during drag motion rather than drag drop, but I'm not sure of the implications.
FWIW, File-Roller suffers from the same issue.

**Comment by zhuyaliang** [comment]
@cwendling I suspect that the problem lies within ```pcmanfm```, and I will try to solve it again. If anyone needs this ```PR``` during this period, I will open it again
Thank you for your code review and feedback. Due to vacation, I was unable to reply to the message in a timely manner

**Comment by zhuyaliang** [comment]
@cwendling I tried to fix the problem in [pcmanfm,(https://github.com/lxde/libfm/pull/103) but I'm not sure if this fix is correct，Please help review the code

**Comment by zhuyaliang** [comment]
There are other issues with ```libfm```, which may not be the most perfect fix, but I don't want to invest too much time in ```libfm```. My focus is on ```mate desktop```

**Pull request closed without merging**